nodeServerless/Local CE SAM/CE_lambda_functions/card_put.js

```diff
@@ -0,0 +1,70 @@
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+
+let response;
+var AWS = require('aws-sdk');
+AWS.config.update({
```

Comment:
It would be nice if you could configure this in one place for all your handlers. Can you set this as an environment variable in your SAM function?

Reply:
This assumes that something is running first, before the lambda function, that would initialize this. Idk about polymorphism in JS but maybe a parent class that deals with it? Not sure where I'd initialize this besides where it is now
(I found this https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst and don't think it would do what you're looking for?)

Reply:
Looking into logging and it looks like I have to create and configure my logger in every file as well?